apps/nextjs-website/src/lib/cmsApi.ts

```
@@ -176,24 +199,42 @@ export const getSolutionProps = async (
   solutionsSlug: string,
   jsonMetadata?: JsonMetadata
 ) => {
-  const strapiSolutions = await fetchSolution(solutionsSlug)(buildEnv);
+  // TODO: restore this when Strapi will manage guides metadata
+  const strapiSolutions = (await fetchResponseFromCDN(
+    getSyncedSolutionsResponseJsonPath()
+  )) as StrapiSolutions | undefined;
   if (!strapiSolutions || strapiSolutions.data.length < 1) {
     // eslint-disable-next-line functional/no-throw-statements
-    throw new Error('Failed to fetch data');
+    throw new Error('Failed to fetch solution data');
+  }
+  const solutions = makeSolutionsProps(strapiSolutions);
+  const solution = solutions.find((s) => s.slug === solutionsSlug);
+  if (!solution) {
+    // eslint-disable-next-line functional/no-throw-statements
+    throw new Error('Failed to fetch solution data');
```

This Error and the one al row 208 should show a different message.
Here you should throw an error and explain that there is no match between the slug and the fetched solutions